### PR TITLE
[kunlunxin] fix operator correctness and expand coverage

### PR DESCRIPTION
**Pull request by RobertLuobo**
Add and register Kunlunxin-specific operator implementations for numerical,
  attention, padding, reduction, indexing, and fused workloads. Replace
  XPU-incompatible Triton paths and unstable intrinsics with XPU-compatible
  fallbacks to address compiler failures, atomic update loss, and numerical
  mismatches.

### Progress

- [ ] Change is properly reviewed (1 reviewer required, 2 recommended).
- [ ] Change is responded to an issue.
- [ ] Change is fully covered by a UT.

### CODE REVIEW

File: src/flag_gems/runtime/backend/_kunlunxin/ops/attention.py [hunk 1]
```diff
@@ -1095,6 +1216,40 @@ def flash_attention_forward(
...
+        row = torch.arange(q_len, device=q.device)[:, None]
+        col = torch.arange(k_len, device=q.device)[None, :]
+        relative_pos = (row + k_len - q_len - col).abs()
+        scores = scores - alibi_slopes[..., None, None] * relative_pos
+    lse = torch.logsumexp(scores, dim=-1)
+    attn = torch.softmax(scores, dim=-1)
+    if dropout_p:
+        attn = torch.dropout(attn, dropout_p, True)
+    out = torch.matmul(attn, v).transpose(1, 2)
+    return out, lse, None, None, None
```
Comment (linChunky) [inline src/flag_gems/runtime/backend/_kunlunxin/ops/attention.py:1264]: triton implement replaced & bypassed with torch implement.
